### DISCUSSION TITLE
OBS WebRTC "SetRemoteDescription called with no ice-ufrag" Error

### DISCUSSION THREAD

**Discussion by MattEqualsCoder** [bluenviron/mediamtx#2338] [Questions]
## Which version are you using?

v1.0.3

## Which operating system are you using?

- [x] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [ ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

I'm unable to stream from OBS 30.0.0-beta3 to mediamtx via WebRTC Whip. I receive an error "SetRemoteDescription called with no ice-ufrag".

## Describe how to replicate the issue

1. I downloaded a fresh copy of mediamtx
2. Ran via ./mediamtx
3. I setup OBS following these instructions: 
![image](https://github.com/bluenviron/mediamtx/assets/63823784/cf4403ec-935a-4495-87dd-f9ae3f8e6553)
![image](https://github.com/bluenviron/mediamtx/assets/63823784/e238ff1a-56df-4f22-80cf-c0f8a5a0db17)
4. I receive the following error when I stream:
![image](https://github.com/bluenviron/mediamtx/assets/63823784/69ab0a27-22ac-4bd2-b07b-d452d77cd95a)
5. I get the following message in the mediamtx console: [WebRTC] [session d27b0b4c] closed (SetRemoteDescription called with no ice-ufrag)
6. No stream is available at http://localhost:8889/mystream/

## Did you attach the server logs?

```
2023/09/13 12:16:49 INF MediaMTX v1.0.3
2023/09/13 12:16:49 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2023/09/13 12:16:49 INF [RTMP] listener opened on :1935
2023/09/13 12:16:49 INF [HLS] listener opened on :8888
2023/09/13 12:16:49 INF [WebRTC] listener opened on :8889 (HTTP)
2023/09/13 12:16:49 INF [SRT] listener opened on :8890 (UDP)
2023/09/13 12:16:54 INF [WebRTC] [session d27b0b4c] created by 127.0.0.1:56160
2023/09/13 12:16:54 INF [WebRTC] [session d27b0b4c] closed (SetRemoteDescription called with no ice-ufrag)
^C2023/09/13 12:17:07 INF shutting down gracefully
2023/09/13 12:17:07 INF [SRT] listener is closing
2023/09/13 12:17:07 INF [WebRTC] listener is closing
2023/09/13 12:17:07 INF [HLS] listener is closing
2023/09/13 12:17:07 INF [RTMP] listener is closing
2023/09/13 12:17:07 INF [RTSP] listener is closing
2023/09/13 12:17:07 INF waiting for external commands
```

## Did you attach a network dump?

No, I am currently only testing from the computer itself.

**Comment by aler9** [comment]
Hello, i tested publishing from Ubuntu 22.04 / OBS 30.0.0-beta3 / WHIP to MediaMTX 1.0.3 and in my case it worked flawlessly. Make sure that your PC is able to reach the configured STUN server (by default is stun.google.com) or enable `webrtcICEUDPMuxAddress` as explained in the README:

https://github.com/bluenviron/mediamtx#connectivity-issues

[a comment by an automated account is not shown]
